config/config.exs

```diff
@@ -20,6 +20,7 @@ use Mix.Config
 #
 #     config :logger, level: :info
 #
+config :phoenix, :json_library, Jason
```

Comment:
Is this global? Will it break a user who uses Poison for their own phoenix stuff?

Reply:
Nope, the config will only apply within the Scout project.  Phoenix generates a warning during tests or dev without it:

```
warning: failed to load Jason for Phoenix JSON encoding
(module Jason is not available).

Ensure Jason exists in your deps in mix.exs,
and you have configured Phoenix to use it for JSON encoding by
verifying the following exists in your config/config.exs:

    config :phoenix, :json_library, Jason
```

Reply:
Ok, as long as it's scoped just to us, not overriding customers who install our library.